tests/phpunit/tests/style-engine/styleEngine.php

```diff
@@ -539,22 +539,24 @@ public function data_wp_style_engine_get_styles() {
 			'inline_background_image_url_with_background_size' => array(
 				'block_styles'    => array(
 					'background' => array(
-						'backgroundImage'    => array(
+						'backgroundImage'      => array(
 							'url' => 'https://example.com/image.jpg',
 						),
-						'backgroundPosition' => 'center',
-						'backgroundRepeat'   => 'no-repeat',
-						'backgroundSize'     => 'cover',
+						'backgroundPosition'   => 'center',
+						'backgroundRepeat'     => 'no-repeat',
+						'backgroundSize'       => 'cover',
+						'backgroundAttachment' => 'fixed',
```

Comment:
Tiny nit: for these changes, do we also need to add the ticket number to the above `test_wp_style_engine_get_styles` test?

Reply:
Whoops, missed that. Thanks for catching it. 

Done.